test/nimble_template/addons/variants/phoenix/api/error_view_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule NimbleTemplate.Addons.Phoenix.Api.ErrorViewTest do
+  use NimbleTemplate.AddonCase, async: false
+
+  describe "#apply/2" do
+    test "copies the error view files and remove web layout files", %{
+      project: project,
+      test_project_path: project_path
+    } do
+      in_test_project(project_path, fn ->
+        AddonsApi.ErrorView.apply(project)
+
+        assert_file("lib/nimble_template_web/views/error_helpers.ex")
+        assert_file("lib/nimble_template_web/views/error_view.ex")
+        assert_file("test/nimble_template_web/views/error_helpers_test.exs")
+        assert_file("test/nimble_template_web/views/error_view_test.exs")
+        assert_file("test/support/view_case.ex")
```

Comment:
hi @byhbt as the test description includes `and remove web layout files` but for the assertion part it doesn't explain it clearly we don't see the `web layout files` in the assertion part, should we add some `refute_file web_layout` to make it clear 🤔

Reply:
yeah, i update the test context instead, because as discussed yesterday, we won't overwrite the existing files of `web` code base.